references/product-recommendations-api/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "product-recommendations-v1",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com:kurtkanaskie/product-recommendations-v1.git"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "scripts": {
+    "apigeelint": "node ./node_modules/apigeelint/cli.js -e PO003,PO007,TD002 -s ./apiproxy --profile apigeex -f table",
+    "integration": "node ./node_modules/cucumber/bin/cucumber.js target/test/integration/features"
+  },
+  "devDependencies": {
```

Comment:
do you do any js unit tests? if not then please remove the unneeded dependencies

Reply:
removed